src/index.js

```diff
@@ -163,6 +163,109 @@ class Radar {
       }
     );
   }
+
+  static geocode(query, callback) {
+    const publishableKey = Cookie.getCookie(Cookie.PUBLISHABLE_KEY);
+
+    if (!publishableKey) {
+      if (callback) callback(STATUS.ERROR_PUBLISHABLE_KEY);
+
+      return;
+    }
+
+    const qs = `query=${query}`;
+
+    const host = Cookie.getCookie(Cookie.HOST) || DEFAULT_HOST;
+    const url = `${host}/v1/geocode/forward?${qs}`;
+    const method = 'GET';
+    const headers = {
```

Comment:
these are the current `headers` for the `trackOnce` function as well -- do we need to update these to match our other SDKs? We'd need to add `Content-Type`, `User-Agent`, and `X-Radar-Config`

Reply:
captured offline discussion / future direction in this clubhouse ticket -- https://app.clubhouse.io/radarlabs/story/1018